bin/common.sh

```diff
@@ -1,66 +1,72 @@
 #!/usr/local/env bash
 ENV_DIR=${ENV_DIR:-./env}
 
-readonly otomiSettings="$ENV_DIR/env/settings.yaml"
-readonly clustersFile="$ENV_DIR/env/clusters.yaml"
-readonly helmfileOutputHide="(^\W+$|skipping|basePath=|Decrypting)"
-readonly helmfileOutputHideTpl="(^[\W^-]+$|skipping|basePath=|Decrypting)"
-readonly replacePathsPattern="s@../env@${ENV_DIR}@g"
-readonly otomiToolsImage="otomi/tools:1.4.10"
+readonly otomi_settings="$ENV_DIR/env/settings.yaml"
+readonly clusters_file="$ENV_DIR/env/clusters.yaml"
+readonly helmfile_output_hide="(^\W+$|skipping|basePath=|Decrypting)"
+readonly helmfile_output_hide_tpl="(^[\W^-]+$|skipping|basePath=|Decrypting)"
+readonly replace_paths_pattern="s@../env@${ENV_DIR}@g"
+readonly otomi_tools_image="otomi/tools:latest"
 
-has_docker=$( docker --version &>/dev/null && echo "1" || echo "0")
-dind=0
+has_docker='false'
+if docker --version &>/dev/null ; then
+  has_docker='true'
+fi
```

Comment:
I agree, but @Morriz requested true/false: https://github.com/redkubes/otomi-core/pull/271#discussion_r559456999

Reply:
And although true/false is not explicitly stated in our style guide, not in Google's styleguide, it is used there (second code block): https://github.com/google/styleguide/blob/gh-pages/shellguide.md#case-statement

Reply:
Yes, I wanted to mention this. I reviewed the style guide, which was merged today, and after that it doesn't contain any statement about not using 0 and 1. But I don't need to have the last word on it, @Morriz probably has a good reason to avoid 0 and 1.